js/everything.js

```diff
@@ -357,6 +364,16 @@ function getCurrentVideoID() {
     return v;
 }
 
+function getCurrentTimePosition() {
+    var t = parseInt(getParameterByName(window.location.search, "t"), 10);
+    if (t >= 0 && t < 999999) {
```

Comment:
Should use `if (t > 0 && t < Number.MAX_VALUE)`, we can ignore it if the value is `0` since it's' noop.